cleanlab/rank.py

```diff
@@ -556,52 +556,54 @@ def get_confidence_weighted_entropy_for_each_label(
     return label_quality_scores
 
 
-def get_knn_distance_ood_scores(
-    features: np.array, nbrs: NearestNeighbors, k: int = None
-) -> np.array:
-    """Returns the KNN distance out-of-distribution (OOD) score for each datapoint.
+def get_outlier_scores(features: np.array, knn: NearestNeighbors = None, k: int = 10) -> np.array:
+    """Returns the KNN distance outlier score for each datapoint.
 
-    This is a function to compute OOD scores where higher scores indicate the datapoint is more likely to be OOD.
+    This is a function to compute outlier scores where higher scores indicate the datapoint is more likely to be an outlier.
 
     Parameters
     ----------
     features : np.array
       Feature matrix of shape (N, M), where N is the number of datapoints and M is the number of features.
       This is the "query set" of features for each datapoint which are used for nearest neighbor search.
 
-    nbrs : sklearn.neighbors.NearestNeighbors
+    knn : sklearn.neighbors.NearestNeighbors
       Instantiated NearestNeighbors class object that's been fitted on a dataset in the same feature space.
```

Comment:
NearestNeighbors -> ``NearestNeighbors`` (note the double backticks required for docstring rendering of code-font).

Reply:
Why are we not using `` `` for all the other code-esq stuff in the header? i.e. k=min(10, max_k) -> ``k=min(10, max_k)``

Reply:
It should be using the double backtick for all code-esq stuff in all docstrings.  Sorry the original code PR was merged a bit too quickly, so there is lots of room for improvement in this PR :D 

It'd be helpful if you can go through all the docstrings for this function and properly double backtick the code